PWGHF/vertexingHF/AliAnaTaskSEXic0SL.C

```diff
@@ -0,0 +1,947 @@
+/*
+   Xic0 -> eXi analysis (* newly written, Aug. 2022)
+
+   Chong Kim
+   Pusan National University
+   kimc@cern.ch
+*/
+
+#include "AliAnaTaskSEXic0SL.h"
+ClassImp(AliAnaTaskSEXic0SL);
+
+#include "AliAnalysisManager.h"
+#include "AliAODcascade.h"
+#include "AliAODMCParticle.h"
+#include "AliAODTrack.h"
+#include "AliESDtrack.h"
+#include "AliESDtrackCuts.h"
+#include "AliESDVertex.h"
+
+#include "AliInputEventHandler.h"
+#include "AliMCEvent.h"
+#include "AliMultSelection.h"
+#include "AliNormalizationCounter.h"
+#include "AliPIDResponse.h"
+#include "AliPPVsMultUtils.h"
+#include "AliRDHFCutsXictoeleXifromAODtracks.h" //For fEvtCuts
+#include "AliVEvent.h"
+#include "AliVVertex.h"
+#include "THistManager.h" //For fHisto
+
+#include "TBranch.h"
+#include "TChain.h"
+#include "TDatabasePDG.h"
+#include "TLeaf.h"
+#include "TString.h"
+#include "TTree.h"
+
+#include <cmath>
+#include <iostream>
+#include <vector>
+using namespace std;
+
+AliAnaTaskSEXic0SL::AliAnaTaskSEXic0SL():AliAnalysisTaskSE("AliAnaTaskSEXic0SL")
+{
+	ControlAnaObjects(0);
+	ControlOutputContainers(0);
+	ResetTreeVariables();
+}//Constructor, default
+
+AliAnaTaskSEXic0SL::AliAnaTaskSEXic0SL(const char* name, const char* option):
+	AliAnalysisTaskSE(name), fTaskOpt(option)
+{
+	ControlAnaObjects(0);
+	ControlOutputContainers(0);
+	ResetTreeVariables();
+}//Constructor
+
+AliAnaTaskSEXic0SL::~AliAnaTaskSEXic0SL()
+{
+	ControlAnaObjects(1);
+}//Destructor
+
+void AliAnaTaskSEXic0SL::Terminate(Option_t *)
+{
+	cout <<"\nDone!\n";
+	return;
+}//Terminate
+
+//=======================================================================================
+void AliAnaTaskSEXic0SL::UserCreateOutputObjects()
+{
+	//Eventwise cut, for MB (default)
+	fEvtCutsMB = new AliRDHFCutsXictoeleXifromAODtracks();
+	fEvtCutsMB->SetOptPileup(AliRDHFCuts::kRejectMVPileupEvent); //Multi vertexer pileup rejection
+	fEvtCutsMB->SetTriggerClass("");
+	fEvtCutsMB->SetTriggerMask(AliVEvent::kINT7);
+	fEvtCutsMB->SetUseInt7TriggerPP2012();
+	fEvtCutsMB->SetUsePhysicsSelection(true);
+
+	//Eventwise cut, for HMV0 trigger
+	fEvtCutsHMV0 = new AliRDHFCutsXictoeleXifromAODtracks();
+	fEvtCutsHMV0->SetOptPileup(AliRDHFCuts::kRejectMVPileupEvent);
+	fEvtCutsHMV0->SetTriggerClass("");
+	fEvtCutsHMV0->SetTriggerMask(AliVEvent::kHighMultV0);
+	fEvtCutsHMV0->SetUsePhysicsSelection(true);
+
+	//AliESDTrackCut for track filtering
+	fTrkCuts = new AliESDtrackCuts();
+	fTrkCuts->SetClusterRequirementITS(AliESDtrackCuts::kSPD, AliESDtrackCuts::kBoth);
+	fTrkCuts->SetDCAToVertex2D(true);
+	fTrkCuts->SetMaxChi2PerClusterITS(cut_maxChi2PerClusterITS);
+	fTrkCuts->SetMaxDCAToVertexXY(cut_maxDCAToVertexXY);
+	fTrkCuts->SetMaxDCAToVertexZ(cut_maxDCAToVertexZ);
+	fTrkCuts->SetMinNClustersITS(cut_minNClustersITS); //ITS cluster
+	//fTrkCuts->SetMinNClustersTPC(fNClustersTPCMin = 70); //TPC min # of clusters - why masked?
```

Comment:
1. I'll delete those parts to avoid bias

2. Two cuts you informed me of are being applied during offline selection.
Since the minimum cut values in the offline selection are " # of crossed rows: 65, Ratio: 0.7 ",
I'll set those values as 65 and 0.7.

Reply:
Your edit of today made me think you missed my long reply of two days ago (where I said it was fine to apply them offline). Did you see https://github.com/alisw/AliPhysics/pull/21512#issuecomment-1358985604 ?

Again, please write a new reply instead of editing an existing one

Reply:
"Make sure to apply the same cuts in reco level", yes, I read them through.
The edit today was just a notification for not adding those cuts in RDHF objects,
in case you check the updated code later and wonder why they're not added.